vdp/controller/v1alpha/controller_service.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package vdp.controller.v1alpha;
+
+import "vdp/controller/v1alpha/controller.proto";
+import "vdp/controller/v1alpha/healthcheck.proto";
+
+// Controller service responds to incoming controller requests
+service ControllerPrivateService {
+  // Liveness method receives a LivenessRequest message and returns a
+  // LivenessResponse message.
+  // See https://github.com/grpc/grpc/blob/master/doc/health-checking.md
+  rpc Liveness(LivenessRequest) returns (LivenessResponse);
+
+  // Readiness method receives a ReadinessRequest message and returns a
+  // ReadinessResponse message.
+  // See https://github.com/grpc/grpc/blob/master/doc/health-checking.md
+  rpc Readiness(ReadinessRequest) returns (ReadinessResponse);
+
+  // GetResource method receives a GetResourceRequest message
+  // and returns a GetResourceResponse
+  rpc GetResource(GetResourceRequest) returns (GetResourceResponse);
+
+  // UpdateResource method receives a UpdateResourceRequest message
+  // and returns a UpdateResourceResponse
+  rpc UpdateResource(UpdateResourceRequest) returns (UpdateResourceResponse);
+
+  // DeleteResource method receives a DeleteResourceRequest message
+  // and returns a DeleteResourceResponse
+  rpc DeleteResource(DeleteResourceRequest) returns (DeleteResourceResponse);
+}
```

Comment:
Can we follow how `pipeline/v1alpha/pipeline_public_service.proto` defines the service using the Google API annotations like `google.api.http` and `google.api.method_signature` to pin down the endpoint details?